react/features/base/jwt/middleware.js

```diff
@@ -183,11 +282,24 @@ function _setJWT(store, next, action) {
             action.issuer = iss;
             if (context) {
                 action.callee = context.callee;
-                action.caller = context.user;
+                action.caller = _normalizeCallerFields(context.user);
                 action.group = context.group;
                 action.server = context.server;
+
+                if (action.caller) {
+                    _overwriteLocalParticipant(store, action.caller);
+                }
             }
         }
+    } else if (!jwt && !Object.keys(actionPayload).length) {
+        const jwtState = store.getState()['features/base/jwt'];
```

Comment:
That's `const { caller } = store.getState()['features/base/jwt'];` because `jwtState` is not needed for anything else.